python/lsst/daf/butler/cli/cmd/_remove_runs.py

```diff
@@ -0,0 +1,104 @@
+# This file is part of daf_butler.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+import click
+from typing import Mapping, Sequence
+
+from ..utils import (
+    ButlerCommand,
+)
+from ..opt import (
+    collection_argument,
+    confirm_option,
+    options_file_option,
+    repo_argument,
+)
+from ... import script
+
+
+# messages emitted by remove-runs, defined separately for use in unit
+# tests.
+noRunCollectionsMsg = "No RUN collections were found."
+willRemoveRunsMsg = "The following RUN collections will be removed:"
+willRemoveDatasetsMsg = "The following datasets will be removed:"
+didRemoveRunsMsg = "The following RUN collections were removed:"
+didRemoveDatasetsMsg = "The following datasets were removed:"
+removedRunsMsg = "Removed collections"
+abortedMsg = "Aborted."
+noRepoFoundMsg = "No repo was found at '{root}'"
```

Comment:
Maybe say "No Butler repository was found at..."? What do we say in all the other command lines?  Shouldn't this be a standard message?

Reply:
We don't do this checking in other places (yet?) so there's no standard. Maybe we should just let it fall through. More in the comment at line 85...